distributions/Dist_cmr653.py

```diff
@@ -2,21 +2,20 @@
 from .base_distribution import BaseDistribution
 
 class Dist_cmr653(BaseDistribution):
-	""" the inherited class"""
 	def __init__(self):
 		self.f_max = 1
```

Comment:
this will work, but I think your f_max is too big. It should be the value of your PDF at 2 or -2

Reply:
I will merge now, but you might want to adjust f_max and send a new PR